perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -242,6 +245,12 @@
 # benchmark. This is the filename.
 OBJECTS_WRITTEN_FILE = 'pkb-objects-written'
 
+# If the period of time when some streams, but not all, are active is
```

Comment:
Can you simplify this comment?

From the beginning of the test to the time when all streams are started and active, we call it the "Gap", if the Gap is above a threshold of total test time, we log a warning, because --- (fill your reason here)